### PR TITLE
Add train link for models/intel/machine-translation

### PR DESCRIPTION
**Pull request by Wovchena**
Close https://github.com/openvinotoolkit/open_model_zoo/issues/3568

### CODE REVIEW

**Comment by Wovchena** [comment]
@andreyanufr, please, take a look

File: models/intel/machine-translation-nar-de-en-0002/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Use Case and High-Level Description
 
-This is a Deutsch-English machine translation model based on non-autoregressive Transformer topology.
+This is a Deutsch-English machine translation model based on non-autoregressive Transformer topology. The model is [trained](https://github.com/openvinotoolkit/training_extensions/tree/089de2f24667329a58e8560ed4e01ef203e99def/misc/pytorch_toolkit/machine_translation) on a custom dataset which isn't publicly available.
```
Comment (eaidova) [inline models/intel/machine-translation-nar-de-en-0002/README.md:5]:
```suggestion
This is a Deutsch-English machine translation model based on non-autoregressive Transformer topology. The model is [trained](https://github.com/openvinotoolkit/training_extensions/tree/089de2f24667329a58e8560ed4e01ef203e99def/misc/pytorch_toolkit/machine_translation) on internal dataset.
```

Comment (eaidova) [reply models/intel/machine-translation-nar-de-en-0002/README.md:5]: please apply for others

Comment (Wovchena) [reply models/intel/machine-translation-nar-de-en-0002/README.md]: Applied

**Review by eaidova — APPROVED** [review]

**Review by andreyanufr — APPROVED** [review]

**Pull request merged**